content/applications/productivity/iot/config/troubleshooting.rst

```diff
@@ -99,6 +99,230 @@ Printer`. Follow the steps and select the *make* and *model* corresponding to th
    Epson and Star receipt printers and Zebra label printers do not need a driver to work. Make sure
    that no driver is selected for those printers.
 
+Epson special case
+~~~~~~~~~~~~~~~~~~
+
+When printing a receipt with an Epson printer, the command that is used is called `GS v 0`. However,
+there are some Epson printers models that do not support this command, they include the following
+models:
+
+- TM-U220
+- TM-U230
+- TM-P60
+- TMP-P60II
+
+If the :abbr:`POS (Point of Sale)` system is using one of these printer models, a particular
+process must be undertaken in order to use another command; `ESC *`.
+
+Process to force ESC * command
+******************************
+
+The first step is to check whether the printer is incompatible with `GS v 0` command.
+
+.. seealso::
+   See this `Epson's GS v 0 documentation
+   <https://reference.epson-biz.com/modules/ref_escpos/index.php?content_id=94>`_ for `GS v 0`
+   compatible printers.
+
+Should the printer not be compatible with the `GS v 0` command ensure that it is compatible with the
+`ESC *` command.
+
+.. seealso::
+   See this `Epson ESC * documentation
+   <https://reference.epson-biz.com/modules/ref_escpos/index.php?content_id=88>`_ for `ESC *`
+   compatible printers.
+
+If the printer is not compatible with the `ESC *` command then the following process is
+not possible. Should the printer be compatible to use the `ESC *` command to print, follow this
+process to configure the printer with the :abbr:`IoT (Internet of Things)` box.
+
+To manipulate the :abbr:`IoT (Internet of Things)` box to use the `ESC *` command to print, go to
+the :abbr:`IoT (Internet of Things)` box homepage. Access the :abbr:`IoT (Internet of Things)` box
+homepage by navigating to :menuselection:`IoT app --> IoT Box`. Then click on the ::guilabel:`IP
+address` and this will redirect the page to the :abbr:`IoT (Internet of Things)` box homepage.
+
+Now click on the :guilabel:`Printers server` button. This will redirect the browser to the *CUPS
+page*. Next, go to :menuselection:`Administration --> Printers --> Add Printer`. Now choose the
+printer that should be modified and click :guilabel:`Continue`.
+
+.. tip::
+   If the name of the printer is still uncertain, take the following steps:
+
+   #. Take note of the listed printers on the *CUPS* page.
+   #. Turn the printer off and refresh the page.
+   #. Now compare the difference with the first list to see which printer disappeared.
+   #. Turn the printer back on and refresh the page again.
+   #. Double check the list again to see if the printer re-appears.
+   #. The printer that disappeared and reappears again on the listed printers is the name of the
+      printer in question.
+
+   .. note::
+      This can be :guilabel:`Unknown` under :guilabel:`Local printers`.
+
```

Comment:
What do you think about adding a new h5 heading here? We could name it something like 

```suggestion
Configure CUPS
*************
```